packages/core/core.ts

```diff
@@ -257,7 +285,19 @@ function errorName(error: unknown): string {
 }
 
 function printError(error: unknown) {
-	let errorMessage = `==${process.pid}== Uncaught Exception: Jazzer.js: `;
+	let errorMessage = `==${process.pid}== `;
+	if (!(error instanceof Finding)) {
+		errorMessage += "Uncaught Exception: Jazzer.js: ";
+	}
+	if (error instanceof Finding) {
+		const stack = error.stack?.split("\n");
```

Comment:
Can the stack manipulation code be moved into the `cleanStack` function?

Reply:
done!